pkg/apimachinery/apis/common/v0alpha1/unstructured.go

```diff
@@ -14,7 +14,7 @@ import (
 type Unstructured struct {
 	// Object is a JSON compatible map with string, float, int, bool, []interface{},
 	// or map[string]interface{} children.
-	Object map[string]any
+	Object map[string]interface{}
```

Comment:
`kube-openapi` generation will fail with this error when `map[string]any` is used, but is fine with `map[string]interface{}` 🙃 

```
Generating openapi code for 1 targets
E1114 08:51:34.777207 1599037 openapi.go:331] Error when generating: Object, Object map[string]any
2024/11/14 08:51:34 OpenAPI code generation error: failed executing generator: some targets had errors: failed to generate map property in github.com/grafana/grafana/pkg/apimachinery/apis/common/v0alpha1.Unstructured: Object: map Element kind Interface is not supported in map[string]any
```